Gemfile

```diff
@@ -89,3 +89,6 @@ gem 'gettext_i18n_rails'
 
 gem 'config', '~> 1.0'
 gem 'terminal-table', '~> 1.8'
+
+# needed by rmt-server-pubcloud
+gem 'jwt', '~> 2.1'
```

Comment:
Is this really required? At least in the changes I could not find anything using `JWT`. Maybe I missed it

Reply:
Okay it's apparently required.